### DISCUSSION TITLE
Exporting attachments

### DISCUSSION THREAD

**Discussion by legaljustus** [Budibase/budibase#1411] [Help]
Hi! 

I'm trying out budibase for the first time to see if it suits our business (legal automation),

The first thing I ran into is that I can't really find a way to export attachment data (files) to an external source. I can't seem to send it via REST (the form component doesn't let me select that field) and I can't send them via automate (doesn't allow this datatype). Is there any way that this can be done (now or maybe in future versions)?

context: I need to be able to export files from budibase to a OCR and get back a response to budibase

**Comment by mjashanks** [comment]
Hi There!

You're right, there is currently no way to do this. This definitely seems like something we should be supporting - makes a lot of sense to me.

Just for reference, how is your OCR expecting the file? e.g. via a REST API using `multipart/form-data`. Does it have public documentation on this?

**Reply by legaljustus** [reply]
Thank you for the quick reply!

We use different OCR's for different projects. But the one with the most documentation is [Google Cloud vision](https://cloud.google.com/vision/docs) and we connect our backend to that using integromat, we send it a URL but I think it also supports the method you mentioned. 

We also sometimes use docparser (they also have [pretty good documentation](https://dev.docparser.com/#import-documents) with a bunch of different ways to send documents) 

We're looking to launch our platform in a couple of weeks and I was/am considering budibase as a alternative to what we now have (wordpress based, a whole lot of plugins). Would you know if/ when you could look into this feature? (Not to rush or demand anything, just need to know if budibase is for me)

**Reply by mjashanks** [reply]
This will not be done in the next few weeks, for sure. We will likely implement this by exposing the file's URL  - though that will need to be authenticated.

We should do this over the next few months, but I'm not entirely sure when - we will be doing some roadmap prioritisation over the next few weeks and can consider it then. 

Shorter term (next month), we are creating a "Code Block" for automation. You will be able to write custom code in here. You should able to accomplish an upload with code.

For now, I'll convert this into an Issue. You can then track the progress.

**Reply by mjashanks** [reply]
Issue: https://github.com/Budibase/budibase/issues/1413
